acceptance/acceptance_test.go

```diff
@@ -161,46 +162,45 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	out = repls.Replace(out)
 	doComparison(t, filepath.Join(dir, "output.txt"), "script output", out)
 
-	for key := range outputs {
-		if key == "output.txt" {
+	for relPath := range outputs {
+		if relPath == "output.txt" {
 			// handled above
 			continue
 		}
-		pathNew := filepath.Join(tmpDir, key)
+		pathNew := filepath.Join(tmpDir, relPath)
 		newValBytes, err := os.ReadFile(pathNew)
 		if err != nil {
 			if errors.Is(err, os.ErrNotExist) {
-				t.Errorf("%s: expected to find this file but could not (%s)", key, tmpDir)
+				t.Errorf("%s: expected to find this file but could not (%s)", relPath, tmpDir)
 			} else {
-				t.Errorf("%s: could not read: %s", key, err)
+				t.Errorf("%s: could not read: %s", relPath, err)
 			}
 			continue
 		}
-		pathExpected := filepath.Join(dir, key)
+		pathExpected := filepath.Join(dir, relPath)
 		newVal := repls.Replace(string(newValBytes))
 		doComparison(t, pathExpected, pathNew, newVal)
 	}
 
 	// Make sure there are not unaccounted for new files
-	files, err := os.ReadDir(tmpDir)
+	files, err := ListDir(tmpDir)
 	require.NoError(t, err)
 
-	for _, f := range files {
-		name := f.Name()
-		if _, ok := inputs[name]; ok {
+	for _, relPath := range files {
+		if _, ok := inputs[relPath]; ok {
 			continue
 		}
-		if _, ok := outputs[name]; ok {
+		if _, ok := outputs[relPath]; ok {
 			continue
 		}
-		t.Errorf("Unexpected output: %s", f)
-		if strings.HasPrefix(name, "out") {
+		t.Errorf("Unexpected output: %s", relPath)
+		if strings.HasPrefix(relPath, "out") {
 			// We have a new file starting with "out"
```

Comment:
maybe unrelated to this PR: This comment string seems inaccurate. Not sure what this refers to.

Reply:
The loop iterates over files that are unaccounted for. If the file starts with "out", it's a new output file.